### PR TITLE
RT2.7-Adding interface passive state deviation

### PR DESCRIPTION
**Pull request by divyarema**
Added deviation to skip interface passive state check

### CODE REVIEW

File: internal/deviations/deviations.go [hunk 1]
```diff
@@ -625,3 +625,9 @@ func ISISLspMetadataLeafsUnsupported(dut *ondatra.DUTDevice) bool {
 func QOSQueueRequiresID(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosQueueRequiresId()
 }
+
+// IsisInterfacePassiveStateUnsupported returns true for devices that do not
+// support interface passive state.
+func IsisInterfacePassiveStateUnsupported(dut *ondatra.DUTDevice) bool {
```
Comment (greg-dennis) [inline internal/deviations/deviations.go:631]: "Isis" -> "ISIS"

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: internal/deviations/deviations.go (hunk 1 above)
Comment (divyarema) [reply internal/deviations/deviations.go]: modified in latest commit

**Comment by mohanasm** [comment]
Closing as this change is no longer required.

**Pull request closed without merging**